client/src/services/content.ts

```diff
@@ -476,6 +476,32 @@ Dit zorgt ervoor dat feedback van gebruikers op de juiste plek wordt opgeslagen.
         outro: `[Terug naar de homepage](/)`
     },
 
+    a11y: {
+        breadcrumbs: `- [Home](/)
+- [Toegankelijkheidsverklaring](/toegankelijkheidsverklaring)`,
+        intro: `# Volledige toegankelijkheidsverklaring
+**De gemeente Amsterdam wil dat iedereen Mijn Amsterdam kan gebruiken. Komt u toch een pagina tegen die u niet kunt lezen of gebruiken? Meld het via ons [contactformulier](https://formulieren.amsterdam.nl/TripleForms/DirectRegelen/formulier/nl-NL/evAmsterdam/Klachtenformulier.aspx).**`,
+        article: `
+### Toegankelijk Mijn Amsterdam
```

Comment:
Ga je hier niet van header 1 in de intro naar header 3? Zou dit dan niet header 2 moeten zijn?